crates/turbopack-ecmascript/src/transform/mod.rs

```diff
@@ -35,6 +35,7 @@ pub enum EcmascriptInputTransform {
     ///
     /// It also provides diagnostics for improper use of `getServerSideProps`.
     NextJs,
+    NextFont,
```

Comment:
We already have `EcmascriptInputTransform::NextJs`, but that seems to be conditionally applied for SSR-ed `pages/*` only. Ideally, this should be flipped: transforms like those for `@next/font` should be part of the NextJs transform that gets applied to all ecmascript code run from next-dev, and we should have a special NextJsSSR transform applied in the above case. What do you think, @alexkirsz?